archivo.py

```
@@ -0,0 +1,7 @@
+# Archivo de prueba python
+
+def prueba():
+  asdasdasd
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)